tests/system/providers/google/cloud/composer/example_cloud_composer.py

```diff
@@ -115,48 +121,14 @@
 
     chain(image_versions, create_env, list_envs, get_env, update_env, delete_env)
```

Comment:
You'll need to add a `TriggerRule.ALL_DONE` to the delete_env cleanup task (outside of the howto block though to keep the docs clean).